src/test/kotlin/org/rust/cargo/toolchain/CargoTest.kt

```diff
@@ -18,19 +18,19 @@ class CargoTest : RsTestBase() {
     fun `test run arguments preserved`() = checkCommandLine(
         cargo.toColoredCommandLine(project, CargoCommandLine("run", wd, listOf("--bin", "parity", "--", "--prune", "archive"))), """
         cmd: /usr/bin/cargo run --color=always --bin parity -- --prune archive
-        env: RUST_BACKTRACE=short, TERM=ansi
+        env: RUSTC=/usr/bin/rustc, RUST_BACKTRACE=short, TERM=ansi
         """, """
         cmd: C:/usr/bin/cargo.exe run --color=always --bin parity -- --prune archive
-        env: RUST_BACKTRACE=short, TERM=ansi
+        env: RUSTC=/usr/bin/rustc, RUST_BACKTRACE=short, TERM=ansi
```

Comment:
Here and in the other tests:

```suggestion
        env: RUSTC=C:/usr/bin/rustc.exe, RUST_BACKTRACE=short, TERM=ansi
```

Reply:
Done, thanks!